components/ILIAS/Mail/classes/class.ilMail.php

```diff
@@ -461,16 +461,18 @@ public function updateDraft(
         string $a_m_subject,
         string $a_m_message,
         int $a_draft_id = 0,
+        ?ilDateTime $a_send_time = null,
```

Comment:
Please use a `DateTimeImmutable` here.

Since it is a future date/time, we MUST NOT store this a UTC. We must use the user timezone and the corresponding date/time string. Storing this as UTC can lead to issues like described here: Mantis Issue: https://mantis.ilias.de/view.php?id=45616#c118361